src/NzbDrone.Core/ImportLists/PTP/CollectionPage.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.ImportLists.PTP;
```

Comment:
Probably best to rename them for code base consistence sake. We can address TMDb later as well.

Reply:
What about `RSSImportList`? Is that different since it's not the entire class name?

Or should this just be `PassThePopcorn`, like its indexer counterpart?

Reply:
`PassThePopcorn` is likely better, good call